### PR TITLE
Update BDLaethc14.xml

### PR DESCRIPTION
**Pull request by gnizla**
a) it's been a while since I did a pull, so want to make sure I am doing everything properly before uploading other stuff.
b) while working on this I was struck by the fact that it would be nice to include references in deco summary, but it's not allowed to use <bibl> is there a way to do this on guidelines?

### CODE REVIEW

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! Everything is fine, I just have one small question. I included the reference (please check that it's the correct one), I didn't get an error message.

File: OxfordBodleian/Juel-Jensen/BDLaethc14.xml [hunk 1]
```diff
@@ -460,73 +460,108 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                          <decoNote type="miniature" xml:id="d3">
+                             <locus target="#61v"/>
+                             <desc> A <term>standing</term>
+                                <ref type="authFile" corresp="AT1013EvaPortrait"/> of <persName ref="PRS6387Luke">Luke</persName> with a hand cross in his right hand and a <term key="codex"/> in the other. He wears an unusual tiara.<q xml:lang="gez">ሥዕለ፡ ሉቃስ።</q>
+                             </desc>
+                          </decoNote>
+
+                          <decoNote type="miniature" xml:id="d4">
+                             <locus target="#100v"/>
+                             <desc> A <term>standing</term>
```
[the remark this answers is not in the document]
Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethc14.xml:518]: I can easily do away with Term thanks

**Comment by gnizla** [comment]
> Thank you! Everything is fine, I just have one small question. I included the reference (please check that it's the correct one), I didn't get an error message.

I don't know, maybe I got an error because of the way I coded it, will amend and then push other stuff, thanks,

**Review by thea-m — APPROVED** [review]
Thank you, Jacopo! Everything seems wonderful to me. I corrected some minor typos and the encoding of term with ana instead of key, which I believe was due to automatic content completion, not intention (let me know if I'm mistaken).

**Comment by gnizla** [comment]
Thank you so much for your prompt reply and work @thea-m, you are a star!

**Pull request merged**